int16[] agent_robo_job
